src/input.rs

```diff
@@ -5,7 +5,44 @@ use std::{
     sync::Arc,
 };
 
-/// All types that are possible arguments to [`cmd!`] have to implement this trait.
+/// All types that are possible arguments to [`cmd!`], [`cmd_unit!`] or
+/// [`cmd_result!`] have to implement this trait.
+/// This makes `cradle` very flexible.
+/// For example passing in an executable can be done as a String,
+/// and passing in a couple of arguments can be done in a [`Vec`]:
```

Comment:
```suggestion
/// and passing in a variable of arguments can be done in a [`Vec`]:
```

Couple is fine, but for slightly formal technical documentation, "a variable number of" might be better.